website/content/docs/templates/hcl_templates/blocks/build/source.mdx

```diff
@@ -1,25 +1,25 @@
 ---
 description: >
-  A source block nested in a build block allows you to use an already defined
-  source and to set specific fields which aren't already set in the top-level
-  source block.
-page_title: source - build - Blocks
+  A `source` block nested in a `build` block reuses a builder defined
```

Comment:
How would you feel about using the sentence you have below instead? It feels more descriptive:
```suggestion
  Add a nested `source` block to your `build` block to reuse a builder defined elsewhere in your configuration.
```